pkg/olog/olog.go

```diff
@@ -118,6 +123,8 @@ func NewWithHandler(h slog.Handler) *slog.Logger {
 
 // SetOutput sets the global logger output to desired writer.
```

Comment:
suggestion: the docs should be updated to mention that there's a mutex on the global logger output.

Reply:
Thanks. Updated the comment.